HW1/P2/P2a.py

```diff
@@ -1,3 +1,29 @@
+import findspark 
+findspark.init()
+
 from P2 import *
+from pyspark import SparkContext
+
+sc = SparkContext()
+
+#create lists of the range for rows and cols  
+ilist = range(2000)
+jlist = range(2000)
+
+#turn the lists into 1 rdd, K: (i,j)
+rddi = sc.parallelize(ilist,10)
+rddj = sc.parallelize(jlist,10)
+rddK = rddi.cartesian(rddj)
```

Comment:
Pay attention to the code redundancy: here you just need one index list mylist = range(2000) and one rdd, you can compute the cartesian product with itself.